### DISCUSSION TITLE
code bug in function  nmc_LGM50_electrolyte_exchange_current_density_Chen2020 ?

### DISCUSSION THREAD

**Discussion by jennya2a** [pybamm-team/PyBaMM#3340] [General]
I tried to build my function based on chen2020:
def nmc_LGM50_electrolyte_exchange_current_density_myfun(c_e, c_s_surf, c_s_max, T):
....
then, i update the parameters using my defined function, but always see an error for line: 
arrhenius = np.exp(E_r / pybamm.constants.R * (1 / 298.15 - 1 / T))
error message:
TypeError: loop of ufunc does not support argument 0 of type PrimaryBroadcast which has no callable exp method

can you help me to fix it? Thanks.

**Comment by chmabaur** [comment]
I guess you should use pybamm.exp instead of np.exp (I think this will change in the future and np functions will be possible to be used).

**Reply by brosaplanella** [reply]
@chmabaur is right, and this should be possible from next release onwards.
